core/state/journal.go

```diff
@@ -290,7 +290,7 @@ func (ch refundChange) dirtied() *libcommon.Address {
 func (ch addLogChange) revert(s *IntraBlockState) {
 	logs := s.logs[ch.txIndex]
 	if len(logs) == 1 {
```

Comment:
There's no need in the `if` because both branches are the same now.

Reply:
no it's 2-dimentional array. renamed `logs` to `txnLogs` for better visibility